src/objectivec/test/model_tests.mm

```diff
@@ -0,0 +1,125 @@
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <XCTest/XCTest.h>
+
+#import "ort_genai_objc.h"
+#import "assertion_utils.h"
+#import <vector>
+#import <array>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ORTGenAIModelTest : XCTestCase
```

Comment:
may want to set [`continueAfterFailure`](https://developer.apple.com/documentation/xctest/xctestcase/1496260-continueafterfailure?language=objc) so that the test logic doesn't keep going after an assertion failure.

Reply:
sure